en/backup-to-gcs.md

```diff
@@ -0,0 +1,208 @@
+---
+title: Back up TiDB Cluster Data to GCS
+summary: Learn how to back up the TiDB cluster to GCS.
+category: how-to
+---
+
+# Back up TiDB Cluster Data to GCS
```

Comment:
Please take a look at https://github.com/pingcap/docs-tidb-operator/pull/19. 
Would it be better to specify the tools in the title? If so, let's make the English titles consistent with the Chinese ones.